koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -270,7 +271,10 @@ def get_expected_nvr(self):
     def format_label(self, label_id):
         """Formats string with user-facing LABEL name and its alternatives"""
 
-        assert label_id in LABEL_NAME_MAP
+        if label_id not in LABEL_NAME_MAP:
+            msg = '"{}" does not contain "{}"'.format(LABEL_NAME_MAP, label_id)
```

Comment:
Maybe we should be more specific in error message that  it's related to label and what does it mean for users.

Could we be consistent about `"` and `'` in error massages at least within one PR? :D

Reply:
> Maybe we should be more specific in error message that it's related to label and what does it mean for users.

I'd love to... what DOES it mean for users (and how is this error different from the one ~L.222)?